.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -159,12 +158,24 @@ jobs:
     - name: Create bundle output directory
       run: mkdir -p definitions/${{ env.SCANNER_BUNDLE_VERSION }}
 
+    - name: Sanity check NVD zip
+      run: |
+        if [ ! -f ./nvd.zip ]; then
+          echo "nvd.zip not found!"
+          exit 1
+        fi
+
     - name: Run Updater (single bundle)
       run: |
+        # Use fully qualified path to refer to the NVD bundle
+        STACKROX_NVD_ZIP_PATH="$(pwd)/nvd.zip"
+        export STACKROX_NVD_ZIP_PATH
```

Comment:
Why do we need the absolute path?

Reply:
I borrowed the idea from a bug that @dcaravel  found.
https://github.com/stackrox/stackrox/pull/12442

Reply:
What's the idea?

Reply:
You can check the description in this PR
https://github.com/stackrox/stackrox/pull/12442

Reply:
I had to look at the description of that PR. I think he used absolute path because the script changes the current directory, which makes the relative path invalid. But that's not the case here, so using an absolute path is not needed. Am I correct or missing something?

Reply:
I agree that `STACKROX_NVD_ZIP_PATH: nvd.zip` probably would work and the absolute path is a bit overkill. But we don’t receive notifications when nvd.zip is missing, and as a result, the bundles are generated without NVD data. To prevent this, I decided to use the absolute path as a precaution.